languages/de.lang

```diff
@@ -331,6 +331,8 @@ $PALANG['pBroadcast_name'] = 'Ihr Name';
 $PALANG['pBroadcast_success'] = 'Ihre Rundmail wurde gesendet.';
 $PALANG['pAdminMenu_broadcast_message'] = 'Rundmail';
 $PALANG['pBroadcast_error_empty'] = 'Die Felder Name, Betreff und Nachricht dürfen nicht leer sein!';
+$PALANG['broadcst_mailboxes_only'] = "Nur an Postfächer senden";
+$PALANG['broadcst_to_domains'] = "Senden an Domains:";
 
```

Comment:
nice typo - please change broadcst_* to broadcast_* ;-) (hint: running language-update.sh helps to find such issues)